### PR TITLE
fix(openai): fix inconsistency in logging for images/edits/audio endpoints

### PR DESCRIPTION
**Pull request by Yun-Kim**
This PR resolves some inconsistencies in logging for the images, edits, and audio endpoints by doing the following:
- Ensuring audio/images endpoints log the filename, not the full file path (for the images endpoints, the file was previously omitted from the log entirely)
- Ensure image endpoints do not log the full b64_json encoded raw image output data, instead log that the output was "returned"
- Add logging for prompt and input parameters for the audio endpoints, not just the completion
- Ensure that the image endpoints include the predefined `dall-e` model name, as this isn't normally used as a request parameter (the image endpoint uses the `dall-e` model for all requests, and does not allow customizing the model)

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed. If no release note is required, add label `changelog/no-changelog`.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Backport labels are set (if [applicable](../docs/contributing.rst#release-branch-maintenance))

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer has explicitly acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment.
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](../docs/contributing.rst#release-branch-maintenance)

### CODE REVIEW

File: releasenotes/notes/openai-fix-logging-extra-endpoints-d6563f7dd1a5828b.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    openai: Resolves some inconsistencies in logs generated by the image and audio endpoints,
+    including filenames, prompts, and not logging raw binary image data.
```
Comment (mabdinur) [inline releasenotes/notes/openai-fix-logging-extra-endpoints-d6563f7dd1a5828b.yaml:5]: Can we make this a bit more explicit. What inconsistencies were addressed?

File: .codespellignorelines [hunk 2]
```diff
@@ -1,2 +1,3 @@
                 input="thsi si a spelilgn imstkae.",
-
+            input="thsi si a spelilgn imstkae.",
```
Comment (mabdinur) [inline .codespellignorelines:2]: nit: do we need to edit this file.

**Review by mabdinur — DISMISSED** [review]
Left some non blocking suggestions. LGTM overall

**Comment by mabdinur** [comment]
Should we backport this fix?

File: .codespellignorelines (hunk 2 above)
Comment (Yun-Kim) [reply .codespellignorelines:2]: Nope, unfortunately this was multiple separate instances of this spelling "error" being used in `test_openai.py` at different indentation levels, and codespell ignores files require exact lines (including whitespace).

**Comment by Yun-Kim** [comment]
> Should we backport this fix?

No need to backport this fix, the openai v2 integration hasn't been released yet.

File: releasenotes/notes/openai-fix-logging-extra-endpoints-d6563f7dd1a5828b.yaml (hunk 1 above)
Comment (Yun-Kim) [reply releasenotes/notes/openai-fix-logging-extra-endpoints-d6563f7dd1a5828b.yaml:5]: The inconsistencies are the `filenames, prompts, and not logging raw binary image data` 😄

**Review by mabdinur — APPROVED** [review]

**Pull request merged**